nautobot/ipam/filters.py

```diff
@@ -101,6 +101,11 @@ class VRFFilterSet(NautobotFilterSet, StatusModelFilterSetMixin, TenancyModelFil
         to_field_name="pk",  # TODO: Make this work with `prefix` "somehow"
         label="Prefix (ID or name)",
     )
+    prefixes = NaturalKeyOrPKMultipleChoiceFilter(
```

Comment:
That was my first [commit](https://github.com/nautobot/nautobot/pull/6551/commits/ff06928a4c657f18b35b2180ad97eba47047d154. ) actually. But this fix does not work because the number of vrfs is not showing up correctly (always zero) in my manual testing. I think in the URL construction, it is filtering by the data model field name instead of any filters specified.

Reply:
May need to also specify `reverse_lookup`?